test/storage/bitpacking_test.cpp

```diff
@@ -0,0 +1,60 @@
+#include <algorithm>
```

Comment:
can probably remove as this is redundant with compression_test but try to add more tests to compression_test